resources/energyplus/ProposedEnergy+.idd

```diff
@@ -44381,6 +44381,336 @@ HeatPump:PlantLoop:EIR:Heating,
         \note quadratic curve = a + b*PLR + c*PLR**2 is typical, other univariate curves may be used
         \note PLR = part load ratio (hot load/steady state capacity)
 
+HeatPump:AirToWater:FuelFired:Heating,
+       \memo The object defines a fuel-fired absorption heat pump based on equation-fit models.
+       \min-fields 31
+  A1 , \field Name
+       \required-field
+       \reference HeatPumpAirToWaterFuelFiredHeatingNames
+       \note Name of the fuel fired absorption heat pump system system
+       \reference-class-name validPlantEquipmentTypes
+       \reference validPlantEquipmentNames
+       \reference-class-name validBranchEquipmentTypes
+       \reference validBranchEquipmentNames
+  A2 , \field Water Inlet Node Name
+       \required-field
+       \type node
+       \note Inlet node name of the water side connection
+  A3 , \field Water Outlet Node Name
+       \required-field
+       \type node
+       \note Outlet node name of the water side connection
+  A4 , \field Air Source Node Name
+       \type object-list
+       \object-list OutdoorAirNodeNames
+       \note This is the air source node name, which is the evaporator side of the heat pump in heating mode.
+       \note Enter the name of an OutdoorAir:Node object.
+  A5 , \field Companion Cooling Heat Pump Name
+       \note The name of the companion HeatPump:AirToWater:FuelFired:Cooling object
+       \note This field is used for a heat pump with switchable heating and cooling mode.
+       \type object-list
+       \object-list HeatPumpAirToWaterFuelFiredCoolingNames
+  A6 , \field Fuel Type
+       \type choice
+       \key NaturalGas
+       \key Propane
+       \key FuelOilNo1
+       \key FuelOilNo2
+       \key Diesel
+       \key Gasoline
+       \key Coal
+       \key OtherFuel1
+       \key OtherFuel2
+       \default NaturalGas
+       \note Fuel Type (NaturalGas, Propane, Gasoline, Diesel etc.)
+  A7 , \field End-Use Subcategory
+       \type alpha
+       \retaincase
+       \default General
+       \note Any text may be used here to categorize the end-uses in the ABUPS End Uses by Subcategory table.
+  N1 , \field Nominal Heating Capacity
+       \autosizable
+       \minimum> 0
+       \units W
+       \note Nominal Heating Capacity in [W] (autosizeable)
+  N2 , \field Nominal COP
+       \type real
+       \minimum> 0
+       \default 1.0
+       \units W/W
+       \note Nominal COP (Coefficient of Performance)
+  N3 , \field Design Flow Rate
+       \autosizable
+       \minimum> 0
+       \units m3/s
+       \note Design Flow Rate in m3/s (autosizeable)
```

Comment:
required-field everywhere, unless otherwise impossible. Not going to keep commenting on all fields.

Reply:
I apologize, clearly I was commenting on the wrong IDD. Ignore this.